internal/controller/services/auth/resources/admingroup-clusterrole.tmpl.yaml

```diff
@@ -2,60 +2,63 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
   name: admingroupcluster-role
+  labels:
+    app.kubernetes.io/managed-by: opendatahub-operator
+    platform.opendatahub.io/part-of: auth
```

Comment:
Do we need to specify this?  wouldn't the template mechanism add these already?

Reply:
should i remove the part-of label?